### PR TITLE
[WIP] Workspace link unlink commands

### PR DESCRIPTION
**Pull request by ajaymathur**
[WIP] 🎢

### CODE REVIEW

File: src/commands/workspace/__tests__/link.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+//@flow
+import { workspaceLink } from '../link';
+import { copyFixtureIntoTempDir } from 'jest-fixtures';
```
Comment (ajaymathur) [inline src/commands/workspace/__tests__/link.test.js]: @lukebatchelor need help with test , 🆘

File: src/cli.js [hunk 2]
```diff
@@ -84,8 +84,10 @@ const commandMap = {
   WORKSPACE: { workspace: true, w: true },
   WORKSPACE_ADD: { add: true },
   WORKSPACE_EXEC: { exec: true },
+  WORKSPACE_Link: { link: true },
```
Comment (lukebatchelor) [inline src/cli.js]: `WORKSPACE_LINK`*

**Comment by lukebatchelor** [comment]
Hmm... So, linking and unlinking should only really apply to external deps right?
Internal deps are already linked and should always remain that way right?

* Attempting to link an internal dep should warn/throw
* Attempting to unlink an internal dep should throw
* Linking an external dep should link at the root level (running yarn link there) then checking that the required symlinks in the packages still exist. (They should already be pointing to the root).
* Unlinking should be the same
* There is an error/warning when symlinking that warns when packages are linked outside the repo, we might need to update that message

**Comment by ajaymathur** [comment]
closing this in favour of https://github.com/boltpkg/bolt/pull/81

**Pull request closed without merging**